cmd/kubeadm/app/preflight/checks.go

```diff
@@ -435,15 +426,20 @@ func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
 	// Run the system verification check, but write to out buffered writer instead of stdout
 	bufw := bufio.NewWriterSize(os.Stdout, 1*1024*1024)
 	reporter := &system.StreamReporter{WriteStream: bufw}
+	runtimeOS := runtime.GOOS
 
 	var errs []error
 	var warns []error
-	// All the validators we'd like to run:
+	// All the common validators we'd like to run:
 	var validators = []system.Validator{
-		&system.OSValidator{Reporter: reporter},
 		&system.KernelValidator{Reporter: reporter},
-		&system.CgroupsValidator{Reporter: reporter},
-		&system.DockerValidator{Reporter: reporter},
+		&system.DockerValidator{Reporter: reporter}}
+
+	if runtimeOS != "windows" {
+		//add non-windows validators
+		validators = append(validators,
+			&system.OSValidator{Reporter: reporter},
```

Comment:
I would expect the system verification check to support windows in this validator; can you fix that?

Reply:
@luxas, I'm not sure about this one.  The current check runs `uname` and verifies that it returns `sysSpec.OS`.  For unix, it verifies it returns `Linux` as the unix variant, as opposed to SunOS, or FreeBSD, etc.  I'm not sure there's an equivalent command or even the same "variant" concept in Windows.